docs/30DaysOfPWA/advanced-capabilities/06.md

```diff
@@ -1,15 +1,170 @@
-| ------- | ----------- |
-| **Definition** | Can I sync data without my PWA being open? |
-| **Differentiation** | What are my options for syncing data? |
-| **Adoption** | What are some good examples of sites syncing data in the background? |
-| **Exercise** | Create a sync task. |
-| **Related** | Look for other advanced capabilities throughout [Week 2: Advanced Capabilities](../advanced-capabilities). |
+## Background services
+
+Modern browsers and Service Worker APIs bring exciting new features to the web. Now, you can run some pieces of your frontend code completely independently from the main application lifecycle. That means you are no longer limited to architecting and building your application as something that only starts when the user opens the corresponding URL and immediately ends right after they close the tab. Now, parts of your code can be executed even when there is no browser window with your application open in it, i.e. in the background. And this can happen with or without notifying the user. It's you, as a developer, who decide.
+
+There are a bunch of APIs forming the **Background Services** feature in Chromium-based browsers which allow you to run code in the background. Today, we'll take a closer look at two that make it possible to synchronize data between your application and the browser: **Background Sync API** and **Periodic Background Sync API**.
+
+## One-off synchronization on restoring the connection
+The web is dependent on internet connectivity, which isn't great as an application platform. Once you’ve downloaded and installed an application, you expect it to work, whether you’re online or offline. The offline-readiness of PWAs is taken care of by using a combination of the core Service Worker API events — `install`, `activate`, `fetch` — and Cache Storage. But what about the Requests that the app does during runtime? How to provide a smooth user experience when there is no internet connection?
+
+The Background Sync API is here to help. It allows developers to think about the frontend app to server communication as a set of _synchronizations_. These "syncs" will happen immediately after they were created ("registered" in terms of Background Sync API) if the connection is available or, if there is no connection, later when the user goes back online. The difference from all other pre-PWA approaches is that "later" can be _after_ the user closed the application tab and/or the visible part of the browser - thanks to the service worker that is always "on duty" in the background.
+
+Please note, this API only provides a `sync` event to the service worker. You still have to preserve the data you want to send (in case the device is offline) and send it. You’ll need to store the data and implement queuing the data pushes if you want to support multiple transactions in a single sync. For storing the data, consider using IndexedDB API - it's asynchronous so you can access it from your service worker.
```

Comment:
@aarongustafson please review the last sentence about IndexedDB

Reply:
LGTM!